cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -52,9 +52,7 @@ include:
   # Genomio docker deploy (on release tag creation)
   - local: cicd/gitlab/parts/dockerbuild.genomio.gitlab-ci.yml
     rules:
-      - if: '$CI_COMMIT_TAG =~ /v[0-9]+\.[0-9]+\.[0-9]+$/'
```

Comment:
Im pretty sure we need to base tag as we need this to run when we create a standard release via merge to main + create a new release tag

Reply:
On closer inspection, the rebuild portion of this tag can be present or absent right. If so disregard first comment :)

Reply:
The new regex should cover both cases (the `_rebuild_[az]` is surrounded by a `?`). If it is not too clear, happy to divide it in two as it was before.